packages/wrangler/src/hyperdrive/create.ts

```diff
@@ -145,9 +144,9 @@ export async function handler(
 		const origin =
 			args.accessClientId && args.accessClientSecret
 				? {
-						host: url.hostname,
+						host: url.hostname + url.pathname,
```

Comment:
This doesn't seem right? Why is the pathname being passed to `host`?

Reply:
It's a special case, we allow paths on Hyperdrive over Access hostnames, since they represent Cloudflare Tunnel endpoints which can have paths.